website_snippet_country_dropdown/static/src/js/web_tour_country_dropdown.js

```diff
@@ -0,0 +1,35 @@
+odoo.define("website_snippet_country_dropdown.tour_demo_page", function (require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+    var base = require("web_editor.base");
+
+    var country_code_test = 'ES';
+    var vat_number_test = 'B01010101';
+
+    tour.register("demo_page", {
```

Comment:
Don't register the tour with this name. Tours share a global namespace, and `demo_page` seems like you are testing another thing. Better call it `website_snippet_country_dropdown_tour_demo_page` for example.